CHANGELOG.md

```diff
@@ -1,3 +1,63 @@
+# v1.0.5 (2021-01-15)
+
+**Note for aws-ecs-1 variant**: due to a change in the ecs-agent's data store schema, the aws-ecs-1 variant cannot be downgraded after updating to v1.0.5.
```

Comment:
nit: `ecs-agent` is the name of the package but "ECS agent" is the name of the software

```suggestion
**Note for aws-ecs-1 variant**: due to a change in the ECS agent's data store schema, the aws-ecs-1 variant cannot be downgraded after updating to v1.0.5.
```